auth/auth_test.go

```diff
@@ -52,7 +52,7 @@ func TestLoadAuthSchemes(t *testing.T) {
 			t.Fatalf("could not create file on disk %s", err)
 		}
 
-		result, err := LoadAuthSchemes(map[string]config.AuthScheme{
+		result, _ := LoadAuthSchemes(map[string]config.AuthScheme{
```

Comment:
I'm not super familiar with this code path, is ignoring errors OK here?

Reply:
the LoadAuthSchemes() can only throw an error if the type is not "basic".